### ISSUE TITLE
Document how to create/use helper plugins and themes for e2e tests

### ISSUE THREAD

**Issue by nielslange** [woocommerce/woocommerce#45780]
> [!CAUTION]
> This issue is currently blocked by #46125, #46745 and #46746.

### Describe the solution you'd like

At the Rubik team meetup in Amsterdam, we've discussed e2e tests and that we there's no documentation on how to create and use helper plugins and themes for e2e tests. I volunteered to create document this, see pca54o-7bV-p2.

### Describe alternatives you've considered

n/a

### Additional context

n/a

**Comment by timmy5685** [comment]
Hi @nielslange - thanks for the report.

I know this has been open for a long time, and I appreciate your patience. We're currently going through some older WooCommerce issues, identifying what still needs fixing, and tidying up what's already fixed or out of scope.

I've taken a look and passed this over to the Developer Advocacy team.

What I'm seeing: The current E2E README documents helper plugins, but equivalent theme guidance is still not clearly documented even though helper themes exist in the test workspace.

**Comment by bacoords** [comment]
I think this is a question for @woocommerce/rubik - does anyone at Rubik want to prioritize working on this documentation, otherwise I suggest we close the issue and see if the idea resurfaces again.

**Comment by gigitux** [comment]
@bacoords Let's close this issue. With AI, I am not convinced that this is really necessary.
